docs/src/guides/typo3/deploy/customize.md

```diff
@@ -0,0 +1,117 @@
+
+```bash
+$ composer require platformsh/config-reader
+```
+
+### Avoiding deadlock with the Local Page Error Handler
+
+A default TYPO3 installation has a risk of deadlocks when run on low-worker PHP-FPM configurations. Specifically, TYPO3 handles 403 and 404 error pages by issuing a full HTTP request back to itself with no timeout, which can lead to process starvation. There are two required steps to avoid this problem.
+
+First, you will need to install the [Local Page Error Handler](https://extensions.typo3.org/extension/pxa_lpeh/) extension for TYPO with the command:
+
+```bash
+$ composer require pixelant/pxa-lpeh
+```
+
+Second, you will need to add a timeout which will set an HTTP timeout of 3 seconds instead of the default several minutes to a new `public/typo3conf/PlatformshConfiguration.php`. You can see this line in context of the full file in the [configuration](#environment) section below.
+
+```php
+$GLOBALS['TYPO3_CONF_VARS']['HTTP']['timeout'] = 3;
+```
+
+You will still need to enable the `pixelant/pxa-lpeh` extension, which you will be able to do in the [post-install](/guides/typo3/deploy/deploy.md#post-install) steps later in this guide.
+
+### (Optional) Introductory examples
+
+The TYPO3 template for Platform.sh loads a number of example content pages provided by TYPO3 into the site when dependencies are installed pre-build. Their inclusion is completely optional, but a helpful choice if you are creating a new TYPO3 site from scratch using Composer.
+
+```bash
+$ composer require typo3/cms-introduction
+```
+
+## Configuration
+
+You're going to add and/or edit a few files described in the sections below, which will result in the following directory structure:
+
+```bash
+.
+├── config
+│   └── sites
+│       └── main
+│           └── config.yaml
+...
+├── public
+│   └── typo3conf
+│       ├── AdditionalConfiguration.php
+│       └── PlatformshConfiguration.php
+└── src
+    ├── SetupConfiguration.yaml
+    └── SetupDatabase.yaml
+```
+
+### Setup
+
+It's likely that you have a setup configuration file (`src/SetupConfiguration.yaml`) already present in your repository which manages your installation during build when running the `typo3 install:setup` command. On Platform.sh, it is recommended that that file  include at least the following commands.
+
+{{< github repo="platformsh-templates/typo3" file="src/SetupConfiguration.yaml" lang="yaml" >}}
+
+### Site
+
+Similarly, you likely have a `config.yaml` file in your repo, althout it may not be in the `config/sites/main` subdirectory as shown above. This file contains settings for the error handling of your site, the languages it will support, and a few others. For the purposes of this guide, you will need to set the `base` attribute to an environment variable called `PLATFORM_ROUTES_MAIN`. 
```

Comment:
```suggestion
You will have to locate the site configuration file(s), `config.yaml`, in your repository's `config/sites/<SITEID>` subdirectories. For the purposes of this guide, you will need to set the `base` attribute to an environment variable called `PLATFORM_ROUTES_MAIN`. 
```

I think you'll want to show how to do this. The simplest version is line 1 in this file: https://github.com/t3kit/t3kit-starter/blob/master/.localconf/platformsh/config/sites/site2/config.yaml#L1

However, the best practice would be to define a baseVariant for the production environment, along the lines of this file: https://github.com/t3kit/t3kit-starter/blob/master/config/sites/t3kit/config.yaml#L2-L5